src/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -36,95 +36,95 @@
 */
 @isTest
 public with sharing class AFFL_MultiRecordType_TEST {
-	
-	private static ID orgRecTypeID;
-	private static ID householdRecTypeID;
-    
-	public static void setup() {
-		UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe_API.getHhAccRecTypeID()));
-        		
-		List<Affl_Mappings__c> mappings = new List<Affl_Mappings__c>();
-    	mappings.add(new Affl_Mappings__c(Name = 'Business Organization', Account_Record_Type__c = 'Business Organization', Primary_Affl_Field__c = 'Primary Business Organization'));	
-    	mappings.add(new Affl_Mappings__c(Name = 'Household Account', Account_Record_Type__c = 'Household Account', Primary_Affl_Field__c = 'Primary Household'));
-    	insert mappings;
-    	
-    	orgRecTypeID = UTIL_Describe_API.getBizAccRecTypeID();
-    	householdRecTypeID = UTIL_Describe_API.getHhAccRecTypeID();
-	}
-	
-	@isTest
-	public static void createPrimaryAffl() {		
-		setup();
-
-    	Contact contact = new Contact(FirstName = 'Test', LastName = 'Testerson');
-    	insert contact;
-    	
-    	//Verify default household account has been created for the contact
-    	contact = [select Account.ID, Primary_Household__c from Contact where ID =:Contact.ID];
-    	System.assertNotEquals(null, contact.Account.ID);
-    	System.assertEquals(null, contact.Primary_Household__c);
-        
+
+    private static ID orgRecTypeID;
+    private static ID householdRecTypeID;
+
+    public static void setup() {
+        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe_API.getHhAccRecTypeID()));
+
+        List<Affl_Mappings__c> mappings = new List<Affl_Mappings__c>();
+        mappings.add(new Affl_Mappings__c(Name = 'Business Organization', Account_Record_Type__c = 'Business Organization', Primary_Affl_Field__c = 'Primary Business Organization'));	
+        mappings.add(new Affl_Mappings__c(Name = 'Household Account', Account_Record_Type__c = 'Household Account', Primary_Affl_Field__c = 'Primary Household'));
```

Comment:
@MatthewBlanski  Looks like most of the test methods in this class test only when label names are used for 'Account Record type' and 'Contact Primary Affiliation field'. Should we have tests for each combination of label and API? In this PR or may be another WI?

Reply:
Please add this as a technical requirement to the refactor spec for `CON_primaryAffls_TDTM`.

Reply:
added it here https://salesforce.quip.com/qu0aAF7LE4Pu#FPDACAWAveB. Resolving.